app/shared/session/useLoginErrors.ts

```diff
@@ -0,0 +1,49 @@
+import { useEffect } from "react";
+import { useSearchParams } from "react-router-dom";
+
+export const LOGIN_ERROR_CODES = {
```

Comment:
those seem more like auth providers than error codes, because usually error codes map to a specific error

Reply:
renamed